README.md

```diff
@@ -133,13 +143,13 @@ For example:
 
 - `weights` The weights for this model. Weights can be given for different formats, but should otherwise be equivalent.
    - `weight_format` Format of this set of weights. Weight formats can define additional (optional or required) fields. See [supported_formats_and_operations.md#Weight Format](https://github.com/bioimage-io/configuration/blob/master/supported_formats_and_operations.md#weight_format)
-        - `[authors]` a list of authors.
+        - `[authors]` a list of authors. It refers to the ones who trained the model.
```

Comment:
I think `It refers to the ones who trained the model` is not quite correct here, because the optional `authors` field here indicates to who create this set of weights, which could also happened by converting the weights from one format to another (so not by training the model).

Reply:
Sorry, I thought those authors were the ones in the main description as it says
`The authors are the creators of the specifications and the primary points of contact.`. I would expect that the ones who created the specification are also the ones responsible for the weights format. 
Maybe we could agree on a general and more specific description?

Reply:
So the idea is that `authors` in the main one are the ones that created the model and uploaded the `model.yaml` etc.
After that happens someone else might convert the weights to a different format and then add this to the same `model.yaml`. In this case, they should be listed here (because they are different from the "main" authors).

Reply:
(I agee, we should explain this in more detail)